src/com/webonise/carbuilder/IColor.java

```diff
@@ -0,0 +1,7 @@
+package com.webonise.carbuilder;
+
+public interface IColor {
+
+	public String getColor();
+	public int getPrice();
```

Comment:
Just to heads up, all methods in interface are by default public. But its up to you.